packages/react-router-dom/server.tsx

```diff
@@ -200,7 +188,7 @@ function getStatelessRouter(
       throw msg("createHref");
     },
     getFetcher() {
-      throw msg("getFetcher");
+      return IDLE_FETCHER;
```

Comment:
Can';t throw here for SSR of `useFetcher`.  Fetchers are always IDLE during SSR - it will throw if you try to do anything with them though.